src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/model/ResidentSummary.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDate
+
+@Schema(description = "Response object which describes an assessment")
+data class ResidentSummary(
+  @Schema(description = "A unique internal reference for the resident", example = "87320")
+  val residentId: Long,
+
+  @Schema(description = "The resident's forename", example = "Dave")
+  val forename: String,
+
+  @Schema(description = "The resident's surname", example = "Jones")
+  val surname: String,
+
+  @Schema(description = "The resident's phone number", example = "07634183674")
+  val phoneNumber: String? = null,
+
+  @Schema(description = "The resident's relation to the offender", example = "Mother")
+  val relation: String,
+
+  @Schema(description = "The resident's date of birth", example = "Mother")
```

Comment:
**niggle:** tiny thing but think the example here should be a date

Reply:
ahh yes good spot, I thought I fixed that

Reply:
fixed